### PR TITLE
Homework_5_Olha_Lesiuk

### PR DESCRIPTION
**Pull request by OlhaLesiuk**

### CODE REVIEW

File: Olha_Lesiuk/task_5.1..py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+names = "john peter brian Morgan Adam Maria bart"
+print(names.upper())
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/task_5.1..py]: only the first letter should be capitalized. Your output is : JOHN PETER BRIAN MORGAN ADAM MARIA BART

File: Olha_Lesiuk/task_5.3.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+original_string = " name=Amanda=sssss&age=32&&salary=1500&currency=euro "
+convert_string = original_string.strip().replace("&&", "&").replace("=sssss", "")
+dictionary = {}
+#convert_string.split("&")
+splited_string = convert_string.split("&")
+for substring in splited_string:
+    k, v = substring.split("=")
+    dictionary[k] = v
+
+print(dictionary)
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/task_5.3.py:10]: Good solution. I like it

File: Olha_Lesiuk/task_5.5.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import re
+text = ''.join(open("/Users/olya/Downloads/text_file/text").readlines())
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/task_5.5.py]: an absolute path is not ok.

Comment (bohdandovbsyh) [inline Olha_Lesiuk/task_5.5.py]: open file without closing it

File: Olha_Lesiuk/task_5.5.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+sentences = re.split(r' *[\.\?!][\'"\)\]]* *', text)
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/task_5.5.py]: The Hubble.Space.Telescope is not a sentence.

Comment (OlhaLesiuk) [reply Olha_Lesiuk/task_5.5.py]: I want to clarify this: "Telescope (often referred to as HST or Hubble) is a space telescope that was launched into low Earth orbit in 1990 and remains in operation.... " So this Is not a sentence?

File: Olha_Lesiuk/task_5.3.py (hunk 2 above)
Comment (OlhaLesiuk) [reply Olha_Lesiuk/task_5.3.py:10]: thanks!

File: Olha_Lesiuk/task_5.5.py (hunk 4 above)
Comment (bohdandovbsyh) [reply Olha_Lesiuk/task_5.5.py]: The Hubble.Space.Telescope (often referred to as HST or Hubble) is a space telescope that was launched into low Earth orbit in 1990 and remains in operation....

This is one sentence

File: Olha_Lesiuk/task_5.1..py (hunk 1 above)
Comment (bohdandovbsyh) [reply Olha_Lesiuk/task_5.1..py]: Why did you resolve it without a fix?

Comment (OlhaLesiuk) [reply Olha_Lesiuk/task_5.1..py]: My bad. I thought that I fixed it locally so should mark up it here as resolved and only after that push a code. Will not do this anymore.

File: Olha_Lesiuk/task_5.5.py (hunk 4 above)
Comment (OlhaLesiuk) [reply Olha_Lesiuk/task_5.5.py]: decided to use if __name__ == '__main__' function

Comment (bohdandovbsyh) [reply Olha_Lesiuk/task_5.5.py]: It still split The Hubble.Space.Telescope  as separate sentences

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**